mernarezk/assignment1/Q1.java

```diff
@@ -0,0 +1,54 @@
+package Assignment1;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Q1 {
+	/**
+	 * Main Method for testing purposes.
+	 */
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+
+		String s = in.nextLine();
+		String t = in.nextLine();
+		System.out.println(isAnagram(s, t));

```

Comment:
optional: Do you think you could add some tests for these solutions? I know you must have tested using the command line, but maybe you could include some of the test cases you've used. If you don't want to write unit tests yet, it is fine for now, you could just add calls like checkEquals(isAnagram("cat", "Cat), true) or checkEquals(isAnagram("foo", "bar"), false) to the main method. You can use something like:
public static void checkEquals(boolean actual, boolean expected) { 
    if (actual != expected) throw new IllegalStateException("Failed check");
}

Reply:
Added JUnit tests file